rdt/transformers/datetime.py

```diff
@@ -49,9 +50,14 @@ class UnixTimestampEncoder(BaseTransformer):
     _min_value = None
     _max_value = None
 
-    def __init__(self, missing_value_replacement='mean', model_missing_values=None,
-                 datetime_format=None, missing_value_generation='random',
-                 enforce_min_max_values=False):
```

Comment:
and this type of init

Reply:
Did not find anything to change this. See above comment